JOSESwift/Sources/RSAKeyCodable.swift

```diff
@@ -62,7 +62,7 @@ extension RSAPublicKey: Decodable {
 
         // Other common parameters are optional.
         var parameters: [String: String] = [:]
-        for key in commonParameters.allKeys {
+        for key in commonParameters.allKeys where key != .X509CertificateChain {
```

Comment:
It would be awesome if you could also ignore the `keyOperations` parameter here and in the private key decoding.

```suggestion
    for key in commonParameters.allKeys where key != .X509CertificateChain && key != .keyOperations {
```

Reply:
Oh, I'm sorry, I must've only read the second half of your comment.